packages/manager/modules/pci/src/projects/project/serving/namespace/infos/attach-registry/attach.module.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+
+import routing from './attach.routing';
+import component from './attach.component';
+import service from '../registry.service';
+
+const moduleName = 'ovhManagerPciProjectServingNamespaceInfosAttachRegistry';
+
+angular.module(moduleName, [
+  'ngTranslateAsyncLoader',
+  'oui',
+  'ovh-api-services',
+  'pascalprecht.translate',
+  'ui.router',
+])
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .component('ovhManagerPciProjectServingNamespaceInfosAttachRegistryComponent', component)
+  .service('OvhManagerPciServingRegistryService', service);
```

Comment:
Service `OvhManagerPciServingRegistryService` is declared 3 times. You could reuse the `OvhManagerPciServingRegistryService` declared in `serving/namespace/infos/infos.module.js` ;)